decidim-admin/app/views/layouts/decidim/admin/_sidebar_menu.html.erb

```diff
@@ -0,0 +1,20 @@
+
+<div class="settings-menu__container">
+  <div class="settings-menu">
```

Comment:
As this was extracted from settings, and the idea is to make it more generic, I'd expect for this to be called sidebar menu (as the file name).

```suggestion
<div class="sidebar-menu__container">
  <div class="sidebar-menu">
```

Reply:
We would need to change the other references to settings in this same file

Reply:
I will handle it separatelly

Reply:
Thanks for Suggestion, it has been fixed as part of [bcba67a](https://github.com/decidim/decidim/pull/11654/commits/bcba67a640612446417a510dcacd08c24968a5f0).